iaso/urls.py

```diff
@@ -113,28 +113,28 @@
 router.register(r"derivedinstances", DerivedInstancesViewSet, basename="derivedinstances")
 router.register(r"mobile/orgunits", MobileOrgUnitViewSet, basename="orgunitsmobile")
 router.register(r"featureflags", FeatureFlagViewSet, basename="featureflags")
-router.register(r"mobile/checkversion", CheckVersionViewSet, basename="checkversion")
+router.register(r"mobile/checkversions?", CheckVersionViewSet, basename="checkversion")
 router.register(r"copyversion", CopyVersionViewSet, basename="copyversion")
 router.register(r"dhis2ouimporter", Dhis2OuImporterViewSet, basename="dhis2ouimporter")
 router.register(r"setupaccount", SetupAccountViewSet, basename="setupaccount")
 router.register(r"tasks/create/orgunitsbulkupdate", OrgUnitsBulkUpdate, basename="orgunitsbulkupdate")
 router.register(r"tasks/create/importgpkg", ImportGPKGViewSet, basename="importgpkg")
 router.register(r"tasks", TaskSourceViewSet, basename="tasks")
 router.register(r"comments", CommentViewSet, basename="comments")
-router.register(r"entity", EntityViewSet, basename="entity")
+router.register(r"entities", EntityViewSet, basename="entity")
 router.register(r"mobile/entities", MobileEntityViewSet, basename="entities")
-router.register(r"entitytype", EntityTypeViewSet, basename="entitytype")
+router.register(r"entitytypes", EntityTypeViewSet, basename="entitytype")
 # At the moment we use the same view set but separate it for the future for when we want to be able to
 # change the format in the future
-router.register(r"mobile/entitytype", EntityTypeViewSet, basename="entitytype")
-router.register(r"bulkcreateuser", BulkCreateUserFromCsvViewSet, basename="bulkcreateuser")
+router.register(r"mobile/entitytypes?", EntityTypeViewSet, basename="entitytype")
+router.register(r"bulkcreateusers", BulkCreateUserFromCsvViewSet, basename="bulkcreateuser")
```

Comment:
Same remark as for checkversion, It's not a list of resources but an action, I would not pluralize